src/sdk/sdk.ts

```diff
@@ -439,9 +441,11 @@ export class FriendlyCaptchaSDK {
             });
             widgetPlaceholderStyle.borderColor = "#f00";
             widgetPlaceholderStyle.fontSize = "12px";
-            widgetPlaceholder.textContent = `Anti-Robot check failed to connect to page or ${originOf(
-              wel.src,
-            )}\nCheck your connection and try again.`;
+            widgetPlaceholder.innerHTML = [
```

Comment:
This may clash with some CSPs (I think we avoid innerHTML in many places for that reason). I looked it up, maybe it's only for inline styles, then it's not an issue here I think (it would require `'unsafe-inline'` for `style-src` :thinking_face: 

It may pose a problem for websites that have trusted types enabled: https://developer.mozilla.org/en-US/docs/Web/API/Trusted_Types_API. We may have to explicitly create all the elements (ugh), unless I'm missing something